### DISCUSSION TITLE
Running iOS on Mac

### DISCUSSION THREAD

**Discussion by nnyx0** [utmapp/UTM#5767] [UTM Help (macOS)] [answered]
I've recently been trying to virtualise iPadOS 17.1 on my M1 2020 Macbook Air, but I haven't been able to get it to work. When you try and drop the .ipsw into the MacOS Virtualise creation screen, it says the .ipsw is corrupt. Even when using "Other", it still doesn't boot. Any ideas?

**Comment by saagarjha** [comment]
Virtualization only supports macOS.

**Reply by nnyx0** [reply]
Then is there any other way to create an iOS VM?

**Reply by F1248** [reply]
Xcode Simulator (Xcode.app/Contents/Developer/Applications/Simulator.app)

**Reply by Edgecrusherr160** [reply, marked as answer]
Apple has developed their mobile devices / iOS from beginning in a very locked down that way. I haven't yet seen a public way to create a iOS VM, and it would be a very complex undertaking. Xcode Simulator is the closest you can get, though likely not what you're looking for.

**Reply by Sunstar16** [reply]
Yes, the Corellium Project can even jailbreak them!

**Reply by ink-splatters** [reply]
[@ChefKissInc](https://github.com/ChefKissInc/Inferno/wiki) guys do amazing things!

You can now emulate `iPhone 11` Running `iOS 14.0b5`
